shared/extension/Behaviour.hpp

```diff
@@ -362,6 +363,36 @@ namespace extension::behaviour {
      */
     struct Done {};
 
+    /**
+     * A reactor subtype that can be used when making a behaviour reactor.
+     *
+     * It exposes the additional DSL words that are added by the Behaviour DSL so they can be used without the need for
+     * using statements
+     */
+    class BehaviourReactor : public NUClear::Reactor {
```

Comment:
This is probably out of the scope of this PR though.

Reply:
It would be another PR

The original idea was that there was a DSL that is the Behaviour DSL that's defined in this file. Then there is a module/algorithm called the Director that provides and implements.

That said, it's quite possible that the language in Behaviour.hpp has been muddied and should be cleared up to remove mentions of the Director (or just name the whole thing Director as you suggested)